CHANGES.rst

```diff
@@ -131,6 +131,11 @@ API Changes
 
 - ``astropy.utils``
 
+  - On systems that do not have ``pkg_resources`` non-numerical additions to
```

Comment:
Yes, you are right. I did tag the original issue as a bug two months ago. So, please move this to v1.3.3 under "Bug Fix". Sorry for the inconvenience.